tests/tests_insights_autoupdate.yml

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Testing insights tags
+  hosts: all
+  gather_facts: true
+  become: true
+
+  tasks:
+    - name: Skip when running with embedded Candlepin
+      meta: end_play
+      when:
+        - lookup('env', 'LSR_RHC_TEST_DATA') | length == 0
+      delegate_to: 127.0.0.1
+      become: false
+
+    - name: Import test data
+      include_vars:
+        file: "{{ lookup('env', 'LSR_RHC_TEST_DATA') }}"
+      delegate_to: 127.0.0.1
+      become: false
+
+    - name: Add autoupdate and register insights
+      include_role:
+        name: linux-system-roles.rhc
+      vars:
+        rhc_insights:
+          autoupdate: true
+          state: present
+
+    - name: Get the auto_update line in file configuration
+      command:
+        grep -ixq "^auto_update=true" /etc/insights-client/insights-client.conf
+      register: test_autoupdate_conf
+      ignore_errors: yes
+
+    - name: Check if auto_update is enabled in config file
+      assert:
+        that: test_autoupdate_conf.rc == 0
```

Comment:
You don't really need this check here - you could omit the `ignore_errors` and `failed_when` in the `Get the auto_update line in file configuration` task - if that task fails then the test will report an error there

Reply:
you are right!